webapp/components/popover_list_members.jsx

```diff
@@ -57,7 +57,11 @@ export default class PopoverListMembers extends React.Component {
         const members = this.props.members;
         const teamMembers = UserStore.getProfilesUsernameMap();
         const currentUserId = UserStore.getCurrentId();
-        const ch = ChannelStore.getCurrent();
+        const ch = ChannelStore.get(this.props.channelId);
+
+        if (!ch) {
```

Comment:
Actually, I'm just going to change this so that the ChannelHeader passes the channel in directly. Currently, the ChannelHeader gets the channel and then passes its id into the popover which then gets the channel again which is rather pointless.